site/data/thirdparty.yaml

```diff
@@ -168,6 +168,10 @@ integrations:
     license: 'Commercial'
  
 training:
+  - name: 'The XSS Rat'
```

Comment:
Should be added in alphabetical order?

Reply:
In this case i added it to the top as its the only free one :)